### PR TITLE
Clean up use_alma flag

### LINKED ISSUE #2542: Clean up "use_alma" flag
On the alma branch, in places where we are using this flag, delete the other code path and remove the flag check.

### PR DESCRIPTION
**Pull request by eliotjordan**
- Cleans up use_alma flag
- Renames voyager_record? to alma_record?

Closes #2542

### CODE REVIEW

File: app/models/concerns/blacklight/solr/document/marc.rb [hunk 1]
```diff
@@ -92,7 +92,7 @@ def to_ctx(format)
         end
 
         # returns true if doc originated from voyager
```
Comment (hackartisan) [inline app/models/concerns/blacklight/solr/document/marc.rb:94]: change `voyager` in comment also?

File: app/views/shared/_announcement.html.erb [hunk 2]
```diff
@@ -1,9 +1,7 @@
 <div class="col-12 alert alert-warning announcement">
   <div class="container">
     <p>
-    <% if Rails.configuration.use_alma %>
-      <h1>Thank you for testing the Catalog Alma QA site!</h2> Please provide feedback via the <a href="https://docs.google.com/forms/d/e/1FAIpQLSdS0FRa8h-7_iCtBeIRGbD2pI7zcqWAmz5hxLFip6dR7If3Nw/viewform">Alma Catalog Testing Feedback Form.</a>
```
Comment (hackartisan) [inline app/views/shared/_announcement.html.erb:5]: I don't think we're quite ready to remove this yet.

**Review by hackartisan — CHANGES_REQUESTED** [review]
dang, git diff generation did not do me many favors here! But it looks good except I don't think we should remove the QA feedback message yet. Excited to delete all this code!!

File: app/views/shared/_announcement.html.erb (hunk 2 above)
Comment (eliotjordan) [reply app/views/shared/_announcement.html.erb:5]: Fixed.

**Comment by eliotjordan** [comment]
@hackartisan Ready for re-review.

**Review by hackartisan — APPROVED** [review]

**Pull request merged**